bitnami/etcd/templates/scripts-configmap.yaml

```diff
@@ -254,14 +254,27 @@ data:
     AUTH_OPTIONS="{{ $etcdAuthOptions }}"
     ETCDCTL_ENDPOINTS="{{range $i, $e := until $replicaCount }}{{ $etcdClientProtocol }}://{{ $etcdFullname }}-{{ $e }}.{{ $etcdHeadlessServiceName }}.{{ $releaseNamespace }}.svc.{{ $clusterDomain }}:{{ $clientPort }},{{ end }}"
     # Remove the last comma "," introduced in the string
-    export ETCDCTL_ENDPOINTS="$(sed 's/,/ /g' <<< $ETCDCTL_ENDPOINTS | awk '{$1=$1};1' | sed 's/ /,/g')"
+    export ETCDCTL_ENDPOINTS="${ETCDCTL_ENDPOINTS%%,}"
+    # prepare space separated list of endpoints
+    export etcdctl_endpoints_list="$(sed 's/,/ /g' <<< $ETCDCTL_ENDPOINTS)"
```

Comment:
I would say we can avoid "remove the last comma..." when using this substitution.